prime-router/src/main/kotlin/Metadata.kt

```diff
@@ -96,6 +96,7 @@ class Metadata : Logging {
         IsValidCLIA(),
         InDateInterval(),
         FilterOutNegativeAntigenTestType(),
+        Intersect()
```

Comment:
This is not needed. These filters are only for COVID 19 and the condition filter will not apply to that. Remove

Reply:
I added it because it was needed for the TranslatorTests to work. reportStreamFilterDefinitions is called in findReportStreamFilterDefinitions which is called from Translator.kt on line 230 to get the filter definition. Is there another way to go about this?

Reply:
You have to find another way of implementing this without needed this code for the legacy pipeline.  The condition filter will only exist for the new pipeline. You just need to make sure no one uses that property for the legacy pipeline.